dissect/target/plugins/os/windows/network.py

```diff
@@ -222,15 +224,32 @@ def _try_value(subkey: RegistryKey, value: str) -> str | list | None:
         return None
 
 
+def _get_config_value(key: RegistryKey, name: str) -> set:
+    value = _try_value(key, name)
+    if not value or value in ("", "0.0.0.0", None, [], ["0.0.0.0"]):
+        return set()
+
+    if isinstance(value, list):
+        return set(value)
+
+    return {value}
+
+
 class WindowsNetworkPlugin(NetworkPlugin):
     """Windows network interface plugin."""
 
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self._extract_network_device_config = lru_cache(128)(self._extract_network_device_config)
+
     def _interfaces(self) -> Iterator[WindowsInterfaceRecord]:
+        """Yields found Windows interfaces used by :class:`NetworkPlugin` ``interfaces`` export func."""
```

Comment:
Same question still, is there no rst thing to reference it directly? I seem to remember a `:func:` or `:method:` thing.

Reply:
I think we correctly get what we want with:
```
:meth:`NetworkPlugin.interfaces() <dissect.target.plugins.general.network.NetworkPlugin.interfaces>`
```

But it's kind of pointless anyway as this docstring never gets rendered because it's prefixed with a `_`...

I don't think the current `:class:` achieves anything, so maybe just change it to the above and merge this.